### DISCUSSION TITLE
Porting GnuCash Turkish UCoA

### DISCUSSION THREAD

**Discussion by BerkhanBerkdemir** [ledgersmb/LedgerSMB#6970] [Ideas]
GnuCash provides uniform chart of accounts (UCoA) that can be ported to LedgerSMB. I would like to lead this effort as much as possible. Following UCoA can ben found in their Git [repository]. There are some items and words need to be remapped into LedgerSMB; therefore, I will ask some community help in a pull request.

Detailed UCoA also can be found in this [page] as Turkish for some detailed explanation for future reference.

  [repository]: https://github.com/Gnucash/gnucash/blob/maint/data/accounts/tr_TR/acctchrt_TEKDUZ.gnucash-xea
  [page]: http://www.muhasebedersleri.com/hesaplar/ayrintili-tek-duzen-hesap-plani.html

**Comment by ehuelsmann** [comment]
Please don't hesitate to ask indeed. If you have specific pieces of a proposed change that you have questions about, please open a draft PR and we can discuss there in detail. Issues are a great way to discuss anything before actual changes are being produced -- or after, but more related to approach or process.

**Comment by ehuelsmann** [comment]
@BerkhanBerkdemir i just checked that site, but my Turkish isn't good enough to understand the copyright conditions or even the account descriptions (which is required in order to set the correct options). Would it be an idea that you try to do the port and that i help you either in the chat or through a video call?

**Comment by ehuelsmann** [comment]
Closing: to long without a response; not actionable by the project. Patches welcome.
